te/tedp_docker/Dockerfile

```diff
@@ -53,7 +53,7 @@ RUN apt update && \
 
 # python related requirements
 COPY te/tedp_docker/requirements.txt /tmp/
-RUN pip3 install --upgrade "pip < 21.0"
+RUN pip3 install --upgrade "pip == 22.3.1"
```

Comment:
Should we pin pip to a version? Again this was done to avoid package breakage in pip specific to ubuntu 16 getting EOL. Is it needed?

Reply:
ACK.